doc/ssl/SSL_CTX_set_ct_validation_callback.pod

```diff
@@ -2,39 +2,87 @@
 
 =head1 NAME
 
+SSL_ct_enable, SSL_CTX_ct_enable, SSL_ct_disable, SSL_CTX_ct_disable,
 SSL_set_ct_validation_callback, SSL_CTX_set_ct_validation_callback,
-SSL_get_ct_validation_callback, SSL_CTX_get_ct_validation_callback -
+SSL_ct_is_enabled, SSL_CTX_ct_is_enabled -
 control Certificate Transparency policy
 
 =head1 SYNOPSIS
 
  #include <openssl/ssl.h>
 
- int SSL_set_ct_validation_callback(SSL *s, ct_validation_cb callback, void *arg);
- int SSL_CTX_set_ct_validation_callback(SSL_CTX *ctx, ct_validation_cb callback, void *arg);
- ct_validation_cb SSL_get_ct_validation_callback(const SSL *s);
- ct_validation_cb SSL_CTX_get_ct_validation_callback(const SSL_CTX *ctx);
+ int SSL_ct_enable(SSL *s, int vtype);
+ int SSL_CTX_ct_enable(SSL_CTX *ctx, int vtype);
```

Comment:
Changing the `int` parameters to the enum type would be an improvement, but otherwise these new functions are a nice usability improvement. It certainly makes it more obvious how to simply enable CT, though requiring only a single SCT is arguably not a great policy. Perhaps strict mode should be changed to require two SCTs. That's still not great, but it's better than one.